docs/resources/worker_script.md

```diff
@@ -83,6 +83,8 @@ resource "cloudflare_worker_script" "my_script" {
 - `secret_text_binding` (Block Set) (see [below for nested schema](#nestedblock--secret_text_binding))
 - `service_binding` (Block Set) (see [below for nested schema](#nestedblock--service_binding))
 - `webassembly_binding` (Block Set) (see [below for nested schema](#nestedblock--webassembly_binding))
+- `logpush` (Boolean) Whether to enable [Logpush](https://developers.cloudflare.com/workers/platform/logpush/)
```

Comment:
this is a generated artifact. check out https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/contributing/registry-documentation.md for how to update these files.